src/components/SignupConfirmation.tsx

```diff
@@ -0,0 +1,161 @@
+import React, {useState} from 'react';
+import {Button, Checkbox, Form, Input, Radio} from 'antd';
+
+const SignupConfirmation: React.FC = () => {
+    const onFinish = (values: any) => {
+        // send data to redux
+    };
+
+    const inputStyle = {
+        borderRadius: '5px',
+        height: '48px',
+        backgroundColor: '#E5E5E5',
+        marginBottom: '15px',
+        marginTop: '6px',
+        fontSize: '18px',
+    };
+
+    const radioStyle = {
+        display: 'block',
+        height: '30px',
+        lineHeight: '30px',
+    };
+
+    return (
+        <Form
+            name="basic"
+            layout="vertical"
+            initialValues={{ remember: true, }}
+            onFinish={onFinish}
+            requiredMark={false}
+        >
+            <div style={{marginTop: '36px',
+                fontSize: '18px',}}>
+                <Form.Item
+                    name="check1"
+                    rules={[{ required: true, message: 'Please check all boxes' }]}
+                    style={{ display: 'inline-block', }}
+                >
+                    <Checkbox>
+                        No family member or attendee will visit The Sajni Center if they have been sick in the
+                        past 24 hours, are feeling ill, have been exposed to a virus, or do not have the appropriate
+                        age required immunizations.
+                    </Checkbox>
+                </Form.Item>
+
+                <Form.Item
+                    name="check2"
+                    rules={[{ required: true, message: 'Please check all boxes' }]}
+                    style={{ display: 'inline-block'}}
+                >
+                    <Checkbox>
+                        All parents are to remain at The Sajni Center during programs
+                    </Checkbox>
+                </Form.Item>
+
+                <Form.Item
+                    name="check3"
+                    rules={[{ required: true, message: 'Please check all boxes' }]}
+                    style={{ display: 'inline-block'}}
+                >
+                    <Checkbox>
+                        My children are up to date on all vaccinations and I will provide a copy of my children’s
+                        immunization records prior to attending any programs. (Children who are being treated for cancer
+                        often have severely compromised immune systems, so we are required to collect this
+                        information for their safety. All medical information will be stored in a HIPPA-compliant manner.)
+                    </Checkbox>
+                </Form.Item>
+            </div>
+
+            <div style={{marginTop: '36px',
```

Comment:
We avoid inline styling if possible! Again, we really want to keep all our CSS in one place! I reccomend moving this into a less file if possible.